### ISSUE TITLE
Private Land Mobile in 11.3MHz incorrect

### ISSUE THREAD

**Issue by isaacgerg** [Arrin-KN1E/SDRsharpbandplans#13]
I believe this should be MWARA. https://wiki.radioreference.com/index.php/MWARA

This system is for aircraft voice communications.

**Comment by Arrin-KN1E** [comment]
Thanks for bringing this to my attention! I will fix this soon.

**Comment by Arrin-KN1E** [comment]
@isaacgerg It has been corrected. As a related question; would you be interested in having the listed allocations for NAT family added to the bandplan?

**Comment by isaacgerg** [comment]
Probably wouldn't be a bad idea but i wouldnt kill yourself over it if it
doesnt work smoothly.

**Comment by Arrin-KN1E** [comment]
@isaacgerg Yeah, would have to test it to see. Since I'm (clearly) unfamiliar with aeronautical HF I don't know how frequently they are listened to. But, if there seems to be enough interest and it looks okay, I might add them.
